### DISCUSSION TITLE
High Latency w/ Nova-2

### DISCUSSION THREAD

**Discussion by virtualsapiens** [deepgram/community#965] [General help] [answered]
Since yesterday, we are experiencing huge latency with pre-recorded audio API.  Transcriptions that used to take 5 seconds are taking over a minute sometimes.  I know there was a status update yesterday w/ some issues, but it says resolved. Is there still a known issue?

**Comment by deepgram-community** [comment]
Hey there! It looks like you haven't connected your GitHub account to your Deepgram account. You can do this at https://community.deepgram.com - being verified through this process will allow our team to help you in a much more streamlined fashion.

**Comment by deepgram-community** [comment]
It looks like we're missing some important information to help debug your issue. Would you mind providing us with the following details in a reply?

- The programming language you are working in (e.g. JavaScript, Python).
- The deepgram product you are using (e.g Speech to Text, Agent API)
- A request ID that triggered your error or issue.

**Comment by deepgram-community** [comment, marked as answer]
This is a pretty old post. Sorry we missed getting back to you.

The issues you experienced on this day should be resolved.  For real time status you can always check:  <https://status.deepgram.com/#>

> ![avatar](https://avatars.slack-edge.com/2023-02-20/4855338233600_fad9174023ee3f640ce1_32.png) This message was sent by John Vajda from Deepgram, via our community automation.
